firmware/gameport-adapter/gameport-adapter.ino

```diff
@@ -41,6 +42,8 @@ static Joystick *createJoystick() {
       return new GenericJoystick<4,4>;
     case 0b0100:
       return new CHFlightstickPro;
+    case 0b1100:
```

Comment:
The next free switch code is `0b0110`, also please order the case ascending, right after ThrustMaster.